enos/enos-scenario-agent.hcl

```diff
@@ -32,7 +32,7 @@ scenario "agent" {
       edition = ["ce", "ent", "ent.fips1402"]
     }
 
-    # arm64 AMIs are not offered for Leap 15.4
+    # arm64 AMIs are not offered for Leap
```

Comment:
> arm64 AMIs are not offered for Leap 15.4

This was previously true only for Leap **15.4**, but now there are also no arm64 AMIs available for Leap 15.5 either, so we leave this restriction for Leap generally.